GraphQL-TSC.md

```diff
@@ -133,13 +133,13 @@ The current voting members of the GraphQL TSC are:
 | [Rob Zhu](https://github.com/robzhu)               | AWS                   | Nov 1, 2020 | Dec 31, 2022 |
 | [Sasha Solomon](https://github.com/sachee)         | Twitter               | Nov 1, 2020 | Dec 31, 2022 |
 
-### Becoming a TSC Voting Member
+### Becoming a TSC Member
 
-The [Technical Charter](https://github.com/graphql/foundation/blob/master/GraphQL%20Specification%20Membership%20Agreement%20February%202019.pdf) describes the composition of the TSC. The GraphQL TSC has 10 elected members, plus the Executive Director. TSC members serve a two-year term. Each year, half of the TSC voting seats will be up for election. There are no term limits.
+The [Technical Charter](https://github.com/graphql/foundation/blob/master/GraphQL%20Specification%20Membership%20Agreement%20February%202019.pdf) describes the composition of the TSC. The GraphQL TSC has 10 elected members, plus the Executive Director. TSC members serve a two-year term. Each year, half of the TSC membership seats will be up for election. There are no term limits.
```

Comment:
> TSC members serve a two-year term. Each year, half of the TSC membership seats will be up for election. There are no term limits.

I think you intended to remove "no term limits"?

Reply:
Good feedback. I'm happy to add that atop the other editorial changes.